src/Donors/ServiceProvider.php

```diff
@@ -55,6 +57,10 @@ public function boot()
 
         $this->addCustomFieldsToDonorDetails();
         $this->enforceDonorsAsUsers();
+
+        give(MigrationsRegister::class)->addMigrations([
+            addPhoneColumn::class,
```

Comment:
```suggestion
            AddPhoneColumn::class,
```

Reply:
Resolved in 078a7d30bf7b851381f77f1b4785a2a5e0315de7